code/modules/mob/thermoregulation.dm

```diff
@@ -65,7 +65,7 @@
 	if(loc_temp < bodytemperature)
 		// We're going to try and just use exposed area(temperature difference)/cold divisor, and assume we're only conducting.
 		var/thermal_loss = (1-get_cold_protection())  				// How much of your skin is exposed.
-		if(environment.total_moles > MOLES_CELLSTANDARD || !IS_SPACE_COLD)
+		if((environment.total_moles > MOLES_CELLSTANDARD || !IS_SPACE_COLD)&&(!istype(src.loc,/obj/machinery/atmospherics)))
 			thermal_loss	*= environment.total_moles/MOLES_CELLSTANDARD	// Multiplied by how many moles are in the environment over 103.934, the normal value of a station. - More moles means more heat transfer, that's basic science.
```

Comment:
Also we should probably just remove this, and pretend that human bodies just convect heat at a rate that depends on the temperature differential and surface area alone